core/templates/dev/head/player/LearnerViewBreadcrumb.js

```diff
@@ -89,7 +89,8 @@ oppia.controller('LearnerViewBreadcrumb', [
 
             $scope.averageRating = (
               RatingComputationService.computeAverageRating(expInfo.ratings));
-            $scope.contributorNames = expInfo.contributor_names;
+            $scope.contributorNames = Object.keys(
```

Comment:
Yes, that makes sense.
https://github.com/526avijitgupta/oppia/commit/2335310ba5d37391ee980399902475ea637bb69c